src/components/InputComponent/InputComponent.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import { IInputProps } from './inputComponent.types.d';
+import './styles.scss';
+
+export const InputComponent: React.FC<IInputProps> = ({
+  border,
+  dark,
+  disabled,
+  error,
+  errorMessage,
+  inputValueReciever,
+  ...rest
```

Comment:
destructure `className` from it and pass it to the input className with the class name it already has like so:

const inputClassNames = ['input-wrapper__input', className].filter(Boolean).join(' ')

It is needed to add additional classes to the component if needed. please follow this approach building the components

Reply:
https://github.com/igorizviekov/mt-booking_app/pull/24/commits/30f347fa0e90e34b42e8f68f6a5ffbb712b26c2d